src/BSON/functions.c

```diff
@@ -24,19 +24,21 @@
 #include "php_bson.h"
 
 typedef enum {
-   PHONGO_JSON_MODE_LEGACY,
-   PHONGO_JSON_MODE_CANONICAL,
-   PHONGO_JSON_MODE_RELAXED,
+	PHONGO_JSON_MODE_LEGACY,
+	PHONGO_JSON_MODE_CANONICAL,
+	PHONGO_JSON_MODE_RELAXED,
 } php_phongo_json_mode_t;
 
 /* {{{ proto string MongoDB\BSON\fromPHP(array|object $value)
    Returns the BSON representation of a PHP value */
 PHP_FUNCTION(MongoDB_BSON_fromPHP)
 {
-	zval   *data;
-	bson_t *bson;
+	zval*   data;
+	bson_t* bson;
 
-	SUPPRESS_UNUSED_WARNING(return_value_ptr) SUPPRESS_UNUSED_WARNING(this_ptr) SUPPRESS_UNUSED_WARNING(return_value_used) /* We don't use these */
+	SUPPRESS_UNUSED_WARNING(return_value_ptr)
+	SUPPRESS_UNUSED_WARNING(this_ptr)
+	SUPPRESS_UNUSED_WARNING(return_value_used) /* We don't use these */
```

Comment:
If you have a moment, can you check if these are actually still needed? I seem to recall some `PHP_FUNCTION` and `PHP_METHOD` blocks no using them.